Telemetry/RulesTest.cs

```diff
@@ -1,19 +1,61 @@
 // Copyright (c) Microsoft. All rights reserved.
 
+using System;
+using System.Collections.Generic;
 using System.Net;
 using Helpers;
 using Helpers.Http;
+using Helpers.Models.TelemetryRules;
+using Newtonsoft.Json;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Telemetry
 {
+    [Collection("Telemetry Tests")]
     public class RulesTest
     {
         private readonly IHttpClient httpClient;
+        private ITestOutputHelper logger;
 
-        public RulesTest()
+        private const string DEFAULT_CHILLERS_GROUP_ID = "default_Chillers";
+        private const int SEED_DATA_RETRY_COUNT = 5;
+        private const int SEED_DATA_RETRY_MSEC = 10000;
+
+        private const string RULES_ENDPOINT_SUFFIX = "/rules";
+
+        // list of rules to delete when tests are complete
+        private List<string> disposeRulesList;
+
+        public RulesTest(ITestOutputHelper logger)
         {
             this.httpClient = new HttpClient();
+            this.logger = logger;
+
+            Assert.True(this.ValidChillerGroup()); // Make sure device groups have been created by seed data
+
+            this.disposeRulesList = new List<string>();
+        }
+
+        /// <summary>
+        /// Try to delete all created rules upon completion.
+        /// Each unit test should add the id of any rules created
+        /// to the disposeRuleslist.
+        /// </summary>
+        ~RulesTest()
```

Comment:
Let the class implement IDisposable and put the clean up code in Dispose() method.